content/collections/session-replay/en/session-replay-android-plugin.md

```diff
@@ -5,34 +5,59 @@ title: 'Session Replay Android Plugin'
 
 {{partial:partials/session-replay/sr-android-performance}}
 
-Session Replay captures changes to an app's view tree, this means the main view and all it's child views recursively. It then replays these changes to build a video-like replay. For example, at the start of a session, Session Replay captures a full snapshot of the app's view tree. As the user interacts with the app, Session Replay captures each change to the view as a diff. When you watch the replay of a session, Session Replay applies each diff back to the original view tree in sequential order, to construct the replay. Session replays have no maximum length.
+Session Replay captures changes to an app's view tree, this means the main view and all it's child views recursively. It then replays these changes to build a video-like replay. 
+
+For example, at the start of a session, Session Replay captures a full snapshot of the app's view tree. As the user interacts with the app, Session Replay captures each change to the view as a diff. When you watch the replay of a session, Session Replay applies each diff back to the original view tree in sequential order, to construct the replay. Session replays have no maximum length.
+
+{{partial:admonition type="tip" heading="Report issues"}}
+To report issues with Session Replay for Android, see the [AmplitudeSessionReplay-Android GitHub repository](https://github.com/amplitude/AmplitudeSessionReplay-Android).
+{{/partial:admonition}}
 
 ## Before you begin
 
+The method you use depends on the version of the Amplitude Android SDK you use.
+
+{{partial:tabs tabs="Android-Kotlin, Android (maintenance)"}}
+{{partial:tab name="Android-Kotlin"}}
+If you use the current [Android-Kotlin SDK](/docs/sdks/analytics/android/android-kotlin-sdk), use the instructions for the Android Plugin.
+
 Use the latest version of the Session Replay plugin above `{{sdk_versions:session_replay_android_plugin}}`. For a list of all available versions, see [Maven Central](https://central.sonatype.com/artifact/com.amplitude/plugin-session-replay-android/versions).
 
 The Session Replay Android plugin requires that:
 
 1. Your application is Android-based.
 2. You can provide a device identifier to the SDK.
+{{/partial:tab}}
+{{partial:tab name="Android (maintenance)"}}
+If you use the [maintenance Android SDK](/docs/sdks/analytics/android/android-sdk), use the instructions for Android Middleware.
+
+Use the latest version of the Session Replay Middleware above version `{{sdk_versions:session_replay_android_middleware}}`. For a list of available versions, see all [release versions](https://central.sonatype.com/artifact/com.amplitude/middleware-session-replay-android/versions) on Maven Central.
+
+The Session Replay Middleware requires that:
+
+1. Your application is Android-based.
+2. You are using `2.40.1` or higher of the [(maintenance) Amplitude Android SDK](/docs/sdks/analytics/android/android-sdk).
+3. You can provide a device ID to the SDK.
```

Comment:
```suggestion
* You can provide a device ID to the SDK.
```

Reply:
Numbered lists are only used for multi-step procedures. Everything else uses a bulleted list.